.circleci/config.yml

```diff
@@ -4,12 +4,16 @@ orbs:
   slack: circleci/slack@3.4.2
 
 references:
-  shared-environment: &shared-environment
-    CIRCLE_ARTIFACTS: /tmp/artifacts
-    CIRCLE_TEST_REPORTS: /tmp/test_results
-
-  docker-image: &docker-image
-    - image: circleci/node:12.16.2-browsers
+  defaults: &defaults
+    working_directory: ~/wp-calypso
+    docker:
+      - image: circleci/node:12.16.2-browsers
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/artifacts
+      CIRCLE_TEST_REPORTS: /tmp/test_results
+      PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 'true'
+      DETECT_CHROMEDRIVER_VERSION: 'false'
```

Comment:
See https://www.npmjs.com/package/chromedriver#detect-chromedriver-version.
The doc doesn't say it, but by default this feature is enabled. I had to disable it in order to make it use `CHROMEDRIVER_FILEPATH`